esd-group-3/src/java/models/Client.java

```diff
@@ -134,4 +134,39 @@ public Client retrieveClientByUserId(DBConnection dbcon, int id) {
         
         return client;
     }
+    
+    public void editUser(DBConnection dbcon, String setDatabase, String username, String toChange, String updatedValue) {            
```

Comment:
I would maybe have this main function in the User model, then you don't need to define the function on the Client and User models as they will inherit this method from the User class.